style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.menu {
+  background-color: #3c3a39;
+  background-image: url(./image/toggel.svg);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  color: white;
+  padding: 12px;
+  margin: 24px;
+}
+
+nav {
+  background-color: #3c3a39;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 92px;
+  position: absolute;
+  width: 375px;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav.logo {
+  margin-bottom: 2px;
+  margin-left: 12px;
+}
```

Comment:
Kindly make sure that your using the same font size provided in the Figma file. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192105291-ce13dcfd-1489-4487-9e4b-be7ea80136b1.png)

Reply:
[OPTIONAL] Kuddos on making the changes highlighted by the previous reviewer, however, On makking the change, the alignment of this navbar got messed up. Please put this two in a flex container and they will be automatically aligned.

<img width="362" alt="Screenshot 2022-09-24 at 20 11 42" src="https://user-images.githubusercontent.com/48546992/192110627-4c41fcb7-4fc4-4136-ab90-0ae63cc9c433.png">